src/points/PointsHome.tsx

```diff
@@ -84,13 +87,9 @@ export default function PointsHome({ route, navigation }: Props) {
           <Text style={styles.infoCardTitle}>{t('points.infoCard.title')}</Text>
           <Text style={styles.infoCardBody}>{t('points.infoCard.body')}</Text>
         </View>
-        <ActivityCardSection onCardPress={onCardPress} />
+        <ActivityCardSection onCardPress={onCardPress} pointsSections={pointsSections} />
       </ScrollView>
-      <BottomSheet
-        snapPoints={['50%']}
```

Comment:
this is a personal preference but the vertical spaces triggered me, particularly between the bottom of the button and the bottom of the screen. i think it's better to let the bottom sheet size itself (if we need to increase the height, we should add space above the button)